airbnb/airbnb/Sources/Extension/Date+Extension.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Calender+Extension.swift
+//  airbnb
+//
+//  Created by seongha shin on 2022/05/26.
+//
+
+import Foundation
+
+extension Date {
+    func string(_ format: String) -> String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = format
+        return dateFormatter.string(from: self)
+    }
+    
+    func day() -> Int? {
+        Calendar.current.dateComponents([.day], from: self).day
+    }
+    
+    func addDay(_ addDay: Int) -> Date? {
```

Comment:
함수 매개변수 이름은 명사형으로 사용해주세요🙂
 
```suggestion
    func addDay(_ day: Int) -> Date? {
```

Reply:
넵넵 번경하였습니다~